presto-verifier/src/main/java/com/facebook/presto/verifier/Verifier.java

```diff
@@ -176,6 +186,55 @@ public int run(List<QueryPair> queries)
         return failed;
     }
 
+    private int getDefultHashPartitionCount()
+    {
+        String url = config.getTestGateway();
+        String username = Optional.ofNullable(config.getTestUsernameOverride()).orElse(DEFAULT_USER_NAME);
+        String password = Optional.ofNullable(config.getTestPasswordOverride()).orElse(null);
+
+        int defaultHashPartitionCount = 0;
+        int numOfActiveNodes = 0;
+        try (ResultSet result = executeQuery(url, username, password, "select count(*) as NUM_OF_WORKERS from system.runtime.nodes where coordinator=false")) {
+            while (result.next()) {
+                numOfActiveNodes = result.getInt("NUM_OF_WORKERS");
+            }
+        }
+        catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
Use the logger to log down the error. Also, I am not sure if it's OK to just log down the error and use the default value which is 0. Maybe it's fine or it's better to throw the exception out. Any thoughts?